clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/isolated/IsolatedIngestionServer.java

```diff
@@ -719,7 +726,19 @@ private void initializeIsolatedIngestionServer() {
         null);
     storeIngestionService.start();
     storeIngestionService.addIngestionNotifier(new IsolatedIngestionNotifier(this));
-    ingestionBackend = new DefaultIngestionBackend(storageMetadataService, storeIngestionService, storageService);
+
+    int blobTransferPort = serverConfig.getDvcP2pBlobTransferPort();
+    int fileTransferPort = serverConfig.getDvcP2pFileTransferPort();
+    String rocksDBPath = serverConfig.getRocksDBPath();
+
+    blobTransferManager = new NettyP2PBlobTransferManager(
+        new P2PBlobTransferService(blobTransferPort, rocksDBPath),
+        new NettyFileTransferClient(fileTransferPort, rocksDBPath),
+        new DvcBlobFinder(ClientFactory.getTransportClient(clientConfig)));
+    blobTransferManager.start();
+
+    ingestionBackend =
```

Comment:
Enabling II doesn't mean we should enabled P2P functionality. Our current setup is if the P2P is enabled in the version level config, then after the batch job, the host(server or DVC) creates the snapshot. So if we want to bring this to II, we need to check the version flag. Given we are sunsetting II in some cases, we don't have to add these codes to II for simplicity.

plus

you have created the p2p manager in `IsolatedingestionBackend` so I think we don't need to create here on more time